README.md

```diff
@@ -91,7 +91,18 @@ $ ...
 
 ### DPDK build and install
 
-Now build DPDK and export `RTE_SDK` env variable for DPDK app (DPVS).
+1. Using meson-ninja for building DPDK libraries. The `dpdk.mk` checks for presence of libdpdk
+
+```bash
+$ cd dpdk-stable-18.11.11
+$ mkdir [user desired install folder - dpdklib]
+$ mkdir [user desired build folder - dpdkbuild]
+$ meson  -Dprefix=[dpdklib] dpdkbuild
+$ ninja -C dpdkbuild
+$ cd dpdkbuild; ninja install
+```
```

Comment:
I think it's helpful to add `libdpdk.pc` to pkg-config search path.

```
...
$ cd dpdkbuild; ninja install
$ export PKG_CONFIG_PATH=${PKG_CONFIG_PATH:+$PKG_CONFIG_PATH:}$(pwd)/../dpdklib/lib64/pkgconfig/libdpdk.pc
```

Reply:
agreed,